### ISSUE TITLE
Is there any way to define different model parameters for host/guest/arbiter sepreately？

### ISSUE THREAD

**Issue by gxcuit** [FederatedAI/FATE#3507]
**Is your feature request related to a problem? Please describe.**
当我自定义model的时候，有没有方法可以为host、guest、arbiter 分别指定 parameter class？ 目前他们公用一个parameter class，如果我想让host、guest、arbiter处理不同的参数，目前的办法好像只有设置默认值？

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by mgqa34** [comment]
You are right, fate in current version (<=v1.7) does not support defining  different parameter object for different role, to achieve this, all parameters should be included in the same component parameter object, then users can specified the role parameters in component::role:: filed in  job runtime config. 
If many developers or users want this feature, we will take it into consideration in fate v2.0+.
